shared/src/iosMain/kotlin/com/fbada006/shared/utils/ExportHelper.ios.kt

```diff
@@ -0,0 +1,15 @@
+package com.fbada006.shared.utils
+
+import androidx.compose.ui.graphics.ImageBitmap
+import platform.UIKit.*
+import platform.UIKit.UIActivityViewController
+
+actual suspend fun shareImage(imageBitmap: ImageBitmap?) {
+    val uiImage = arrayOf(imageBitmap)
+    val activityItems = listOf(uiImage)
+    val activityController = UIActivityViewController(activityItems = activityItems, applicationActivities = null )
```

Comment:
I think when we trigger an intent to share items on IOS, we need to define the
`applicationActivities`
Currently, the bottom sheet is showing blank because I set this to null.